target/linux/ramips/dts/U25AWF.dts

```diff
@@ -0,0 +1,108 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/{
+	compatible="u25awf","ralink,mt7620n-soc";
+	model="KimaxU-25AWF";
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+		reset {
+			label = "reset";
+			gpios = <&gpio2 3 GPIO_ACTIVE_HIGH>;
+			linux,code = <0x198>;
```

Comment:
Please use the proper KEY_ value here as listed in [the file @mkresin pointed you to](https://github.com/torvalds/linux/blob/275490680c3dede8550f5ee3e59e2a618d2adb19/include/uapi/linux/input-event-codes.h#L476)

See how it's done [here](https://github.com/lede-project/source/blob/master/target/linux/ramips/dts/ZBT-CPE102.dts)